### ISSUE TITLE
switching emulated computer causes a crash

### ISSUE THREAD

**Issue by Selorax** [vpyk/emu80v4#19]
```
$ emu80qt 
pure virtual method called
terminate called without an active exception
Aborted
```

Switching machines sometimes makes the emulator crash. Though I didn't find how to reproduce it reliably. 

I have 4.0.422 built from source.

**Comment by vpyk** [comment]
There was fix for this bug 09.11.2022 after leleasing the version 4.0.222 in commit 9736547. Would you check if this problem persist with actual git code?

**Comment by Selorax** [comment]
I think the bug is fixed, I didn't experience the crashes of this kind since then.
